add CONTRIBUTING.md (#143)
sepehr-rs/Sudoku: CONTRIBUTING.md

> # Contributing Sudoku      

Reviewer: ```suggestion
# Contributing to Sudoku
```

> Thank you for your interest in contributing Sudoku! This guide will help you contribute to make Sudoku better application for community.

Reviewer: ```suggestion
Thank you for your interest in contributing to Sudoku! This guide will help you contribute to making Sudoku a better application for the community.
```

Note 2: By contributing to Sudoku, you MUST comply with the GNOME Code of Conduct (COC).

## Raising an Issue

We RECOMMENDED that you follow the project's ISSUE TEMPLATE when reporting bugs or requesting features to help us gather the necessary information. However, if you don't use the template, please make sure to include:

- A clear and descriptive title
- Steps to reproduce the issue
- Expected versus actual behavior
- Screenshots or logs if applicable
- Your environment (e.g., OS, Sudoku version)

## Pull Requests

- You MAY consider opening an issue before sending PR to avoid conflicts
- MUST fork the repository from the `dev` branch
- Create your feature branch off `dev`.
- When ready, submit your Pull Request (PR) to merge into the `dev` branch.
- Make sure your code follows the project's coding standards and includes relevant tests if applicable
- Ensure your PR description clearly explains the changes and purpose

# Translation
- MUST fork the repository from the `dev` branch.
- Create your feature branch off `dev`.
- When your changes are ready, open a Pull Request targeting the dev branch.

> Thanks for helping improve Sudoku! 🙌

Reviewer: ```suggestion
Thanks for helping make Sudoku better!
```
